pkg/interfaces/contracts/vault/IBatchRouter.sol

```diff
@@ -100,27 +100,31 @@ interface IBatchRouter {
      * @notice Queries a swap operation involving multiple paths (steps), specifying exact input token amounts.
      * @dev Min amounts out specified in the paths are ignored.
      * @param paths Swap paths from token in to token out, specifying exact amounts in
+     * @param sender Address of the real account that called the swap function
```

Comment:
My understanding is we're doing this so that aggregators or other third parties can simulate calls from particular user accounts (e.g., so that user-sensitive hooks will return correct results), in which case the sender would not necessarily be the real caller (as it is with the non-query functions).

So consider something for all these like "Address passed to the operation as the sender; may influence results (e.g., with user-dependent hooks)"

Also, they all say "swap function," even if the actual operation is something else. Addressed above.